x-pack/test/cloud_security_posture_functional/config.agentless.ts

```diff
@@ -24,7 +24,7 @@ export default async function ({ readConfigFile }: FtrConfigProviderContext) {
         ...xpackFunctionalConfig.get('kbnTestServer.serverArgs'),
         `--xpack.fleet.agents.fleet_server.hosts=["https://ftr.kibana:8220"]`,
         `--xpack.fleet.internal.fleetServerStandalone=true`,
-        `--xpack.fleet.enableExperimental.0=agentless`,
+        `--xpack.fleet.agentless.enabled=true`,
```

Comment:
Removing our dependency on enableExperimental will make deciding in Serverless to use either the Agentless API or the default Agent policy solution easier.